### PR TITLE
[releng] Migration 2021-06 - Kitalpha impacts

### PR DESCRIPTION
**Pull request by arnauddieumegard**
Migrate TP to include:
- Kitalpha 6.0
- Capella 6.0
- Nattable 2.0.1
- Nebula 2.3.0

Jarsigner update
Report #196 from master
Bump to 6.0.0

Change-Id: Id11992a29e5204baa9305fb11f58f6dd9e13750b

### CODE REVIEW

File: docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java [hunk 1]
```diff
@@ -465,12 +472,18 @@ private static String buildResourceRepresentationString(Resource resource, Strin
 			});
 		}
 
-		if (eObject instanceof DSemanticDiagram) {
-			stringBuilder.append(CapellaServices.getPathFromElement(((DSemanticDiagram) eObject).getTarget()));
-			stringBuilder.append("#");
-			stringBuilder.append(CapellaServices.getDiagramUid((DSemanticDiagram) eObject));
+		if (eObject != null) {
```
Comment (bzendagui) [inline docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java:475]: The instanceof return false if the object is null. why did you add this null check?

File: docgenhtml/plugins/org.polarsys.capella.docgen/templates/pattern._2twVYKu9EeCWrf9pgx3zjA/method._2twVZau9EeCWrf9pgx3zjA.pt [hunk 2]
```diff
@@ -21,9 +21,9 @@ String logo = ImageHelper.getTypePng(element, projectName, outputFolder);
 <%
 org.polarsys.kitalpha.doc.gen.business.core.extension.intf.DocGenExtensionFactory.newDocGenExtensionEngine().apply("CAPELLA_REQUIREMENTS_EXTENSION", ctx, getParameters(), stringBuffer);
 %>
-
-
-
+<%
```
Comment (bzendagui) [inline docgenhtml/plugins/org.polarsys.capella.docgen/templates/pattern._2twVYKu9EeCWrf9pgx3zjA/method._2twVZau9EeCWrf9pgx3zjA.pt:24]: Is this a cherrypick of a forgotten commit?

Comment (arnauddieumegard) [reply docgenhtml/plugins/org.polarsys.capella.docgen/templates/pattern._2twVYKu9EeCWrf9pgx3zjA/method._2twVZau9EeCWrf9pgx3zjA.pt:24]: This is a cherry pick from a commit done on master (https://github.com/Iulius3456/capella-xhtml-docgen/commit/1f65575f80d92a9fcf88e308831e46682c4b2ca7)

File: docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java (hunk 1 above)
Comment (arnauddieumegard) [reply docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java:475]: Because it may call `stringBuilder.append(CapellaServices.getPathFromElement(eObject));` on null

Comment (arnauddieumegard) [reply docgenhtml/plugins/org.polarsys.capella.docgen/src/org/polarsys/capella/docgen/util/StringUtil.java:475]: l481

**Review by bzendagui — APPROVED** [review]
